index.html

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<meta-charset="utf-8">
+<title>Mac Calculator Clone</title>
+<link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto">
+<link rel="stylesheet" href="./css/style.css">
+<div class="calculator">
+  <div class="calculator-display-container">
+    <div class="calculator-display">
+      0
+    </div>
+  </div>
+  <div class="calculator-keypad">
+    <div class="calculator-row">
+      <div class="calculator-button calculator-click">
```

Comment:
Having an element that contains an immediate descendant of the same class name seems is probably not a good solution. Why did you need to do this?